LilitYeghiazaryan/recursion_6_7_8_10

```diff
@@ -0,0 +1,69 @@
+#include <iostream>
+#include <vector>
+int bigDivider(int num1, int num2) //6
+{
+	if (num1 == num2)
+	{
+		return num1;
+	}
+	if (num1 > num2)
+	{
+		num1 -= num2;
+	}
+	else
+	{
+		num2 -= num1;
+	}
+
+	return bigDivider(num1, num2);
+}
+
+double n_power(double num, int pow) //7
+{
+	if (pow == 1)
+	{		
+		return num;
+	}
+	if (pow % 2 == 0)
+	{
+		return n_power(num, pow / 2) * n_power(num, pow / 2);
+	}
+	if (pow % 2 != 0)
+	{
+		return num * n_power(num, pow - 1);
+	}
+}
+
+void reverse_cout(const std::vector<int> arr, int index) //8
```

Comment:
The name 'reverse' hints to me that your code should print an array from right to left. Is that what it does?

BTW, add `&` to `arr`